docs/pages/desktop-access/discovery.mdx

```diff
@@ -0,0 +1,128 @@
+---
+title: Desktop Discover
+description: Automatically add Teleport Desktop Access to your infrastructure
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="11"
+>
+
+The features documented on this page are available in Teleport `11` and higher.
+
+</Details>
+
+This page docments the Teleport Desktop Access Discovery wizard. This feature, available through the Web UI, makes it easy to create a Windows Active Directory and configure it to accept Desktop Access connections from Teleport.
+
+## Prerequisites
+
+- A server or virtual machine running a Windows Server operating system.
+- A host where you will run the Teleport Desktop Service.
+- <ScopedBlock scope={["oss", "enterprise"]}> This **can not** be the same instance
+  of Teleport running your Auth and/or Proxy service(s). To install Desktop Access into an existing Teleport instance running other services, see the [Getting Started](getting-started.mdx) guide for Desktop Access.</ScopedBlock>
+
+  See [Installation](../../installation.mdx) for details.
+
+## Step 1/2. Install Teleport
+
+### Set up the Teleport Auth and Proxy Services
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+## Step 2/2. Run the discovery wizard
+
+In your web browser, access the teleport Web UI at <ScopedBlock scope={["oss", "enterprise"]}>`teleport.example.com`</ScopedBlock><ScopedBlock scope={["cloud"]}>`mytennant.teleport.sh`</ScopedBlock>. Select "Desktops" from the main menu, then **ADD DESKTOP**:
```

Comment:
I think the intent is more that people will access this by entering the "Access Manager" interface first under the top-right menu.